common/src/main/java/com/wynntils/services/mapdata/providers/builtin/CharacterProvider.java

```diff
@@ -44,6 +44,9 @@ public Stream<MapCategory> getCategories() {
         return PROVIDED_CATEGORIES.stream();
     }
 
+    @Override
+    public void reloadData() {}
```

Comment:
Now there is a bit of conflict between the inflight PRs, but maybe we should re-query Hades for this?

Reply:
Yes, I did it.